app/components/organisms/HeaderWithSearch/index.tsx

```diff
@@ -12,16 +12,46 @@ export default function HeaderWithSearch() {
 	const user = useOptionalUser()
 
 	return (
-		<header className="container py-6">
-			<nav className="flex flex-wrap items-center justify-between gap-4 sm:flex-nowrap md:gap-8">
-				<div>
-					<Link to="/" className="flex w-20 items-center justify-center">
-						<img src={logo} />
+		<header className="mt-0 py-0">
+			<nav className="dark:bg-dark-secondary flex h-26 flex-wrap items-center justify-between gap-4 bg-slate-700 p-6 sm:flex-nowrap md:gap-4">
+				<div className="flex items-center gap-4"></div>
+				<Link to="/" className="flex w-20 items-center justify-center">
+					<img src={logo} />
+				</Link>
+
+				<div className="flex flex-1 items-center justify-center gap-6">
+					<Link
+						to="/about-us"
+						prefetch="intent"
+						className="rounded bg-slate-400 px-4 py-2 font-bold text-white hover:bg-slate-500"
+					>
+						Home
```

Comment:
This label needs updating to 'About Us' to match its URL:

```suggestion
						About Us
```